### PR TITLE
feature: Sort the stored flows returned with `flow_manager/v2/stored_flows`

### LINKED ISSUE #139: Add new index on `flow.priority` and update `_find_flows` to `sort({"flow.priority": -1, "updated_at": 1})`
Add new index on `flow.priority` and update [`_find_flows`](https://github.com/kytos-ng/flow_manager/blob/master/controllers/__init__.py#L200) to `sort({"flow.priority": -1, "updated_at": 1})`

This is driven by https://github.com/kytos-ng/sdntrace_cp/issues/76

### PR DESCRIPTION
**Pull request by gretelliz**
Closes #139 

### Summary

Sort the flows that are obtained with a request of stored flows with [`find_flows`](https://github.com/kytos-ng/flow_manager/blob/3957bdc61af593b536b39313b3e32d0473a6c597/controllers/__init__.py#L202-L204)

### Local Tests

Requiest: 
```
http://127.0.0.1:8181/api/kytos/flow_manager/v2/stored_flows?dpid=00:00:00:00:00:00:00:02&state=installed
```

Response:
```
{
    "00:00:00:00:00:00:00:02": [
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "set_vlan",
                        "vlan_id": 4001
                    },
                    {
                        "action_type": "output",
                        "port": 3
                    }
                ],
                "cookie": 12310228866111668291,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_vlan": 3824,
                    "in_port": 2
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "c7d11070198ebad1796b22c6d215c74b",
            "id": "dee0f288ec9aed48d4ce2d466ebf7e46",
            "inserted_at": "2022-09-22T21:00:44",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:02",
            "updated_at": "2023-03-03T22:26:59"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "set_vlan",
                        "vlan_id": 3824
                    },
                    {
                        "action_type": "output",
                        "port": 2
                    }
                ],
                "cookie": 12310228866111668291,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_vlan": 4001,
                    "in_port": 3
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "e56d326406a2c309f260c20bf6b8ece1",
            "id": "315f61612ffa8bbb438192d42c12c2ca",
            "inserted_at": "2022-09-22T21:00:44",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:02",
            "updated_at": "2023-03-03T22:26:59"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "output",
                        "port": 4294967293
                    }
                ],
                "cookie": 12321848580485677058,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_type": 35020,
                    "dl_vlan": 3799
                },
                "priority": 1000,
                "table_id": 0
            },
            "flow_id": "23bfec0be8fabf35e1e74c2307b3b0a8",
            "id": "187e17beddc51a7ea8f1c084ce8ec1c5",
            "inserted_at": "2022-09-09T18:20:27",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:02",
            "updated_at": "2023-03-03T23:05:55"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "output",
                        "port": 2
                    }
                ],
                "cookie": 0,
                "hard_timeout": 1200,
                "idle_timeout": 360,
                "match": {
                    "in_port": 1
                },
               "priority": 10,
                "table_id": 0
            },
            "flow_id": "1c0149e7675269672abb6858abd87fed",
            "id": "0962e2d6d4959a6a05e60dc564fada88",
            "inserted_at": "2023-03-03T22:25:25",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:02",
            "updated_at": "2023-03-03T22:56:35"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "output",
                        "port": 4294967293
                    }
                ],
                "cookie": 0,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "priority": 0,
                "table_id": 0
            },
            "flow_id": "2e8155af1a85644a166b28deb5af1f79",
            "id": "61c492f52e9e9271b05ba983b60b29b5",
            "inserted_at": "2022-10-10T20:29:41",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:02",
            "updated_at": "2023-03-03T22:26:59"
        }
    ]
}
```

Note that now the flows are ordered by `priority`.

### End-to-End Tests

Added `test_017_install_flows_and_retrieve_sorted` into `test_e2e_20_flow_manager.py`. Three flows are installed and it is verified that they are returned in descending order by `priority`.

+ python3 -m pytest tests/test_e2e_20_flow_manager.py::TestE2EFlowManager::test_017_install_flows_and_retrieve_sorted
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-7.2.0, pluggy-1.0.0
rootdir: /tests
plugins: timeout-2.1.0, rerunfailures-10.2
collected 1 item

tests/test_e2e_20_flow_manager.py .                                      [100%]

=============================== warnings summary ===============================
test_e2e_20_flow_manager.py: 17 warnings
  /usr/lib/python3/dist-packages/mininet/node.py:1121: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    return ( StrictVersion( cls.OVSVersion ) <

test_e2e_20_flow_manager.py: 17 warnings
  /usr/lib/python3/dist-packages/mininet/node.py:1122: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    StrictVersion( '1.10' ) )

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
------------------------------- start/stop times -------------------------------
================== 1 passed, 34 warnings in 143.86s (0:02:23) ==================

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
@gretelliz, functionality-wise looks good, great to have the e2e test too. 

- Can you update the changelog? This should be in a `Changed` section, since we're changing the behavior of the API response, but this isn't considered breaking compatibility. 

- Can you also add an unit test for this? You can reuse this part here to ensure that `self.flow_controller.db.flows.find.sort` is also called https://github.com/kytos-ng/flow_manager/blob/master/tests/unit/test_flow_controller.py#L158, e2e test will cover that, but primarily let's ensure that our unit tests also cover it, just so if a developer months/years down the road change this they can catch at the unit test level as well.

File: openapi.yml [hunk 1]
```diff
@@ -160,7 +160,7 @@ paths:
           schema:
             type: array
             items:
-              type: int
+              type: integer
```
Comment (viniarck) [inline openapi.yml:163]: Great catch, thanks for fixing this

**Comment by gretelliz** [comment]
Good point @viniarck. I added a [new unit test](https://github.com/kytos-ng/flow_manager/blob/62f1c36a24d30e5c46e2bacb59ee872f9960b6ef/tests/unit/test_flow_controller.py#L166-L171)

**Comment by gretelliz** [comment]
Thank you.

**Pull request merged**